src/Toolkit/Xml.php

```diff
@@ -421,7 +421,7 @@ public static function value($value): ?string
             return $value;
         }
 
-        $encoded = htmlentities($value);
+        $encoded = htmlentities($value, ENT_COMPAT);
```

Comment:
I think this can be:

```suggestion
        $encoded = htmlentities($value, ENT_NOQUOTES | ENT_XML1);
```

as quotes inside XML text are fine, just `<`, `>` and `&` are special.

Reply:
My understanding was that COMPAT was the old default, so making that explicit seemed the easiest/least risky.

Reply:
Good thinking, but I think `NOQUOTES` is the better choice in the long run. Otherwise we will stumble across this later and wonder why we went with `ENT_COMPAT`.